vega_sim/environment/environment.py

```diff
@@ -518,11 +534,11 @@ def step(self, vega: VegaServiceNetwork) -> None:
             try:
                 agent.step(state)
             except Exception as e:
-                msg = f"Agent '{agent.key_name}' failed to step. Error: {e}"
+                msg = f"Agent '{agent.name()}' failed to step. Error: {e}"
                 if self.raise_step_errors:
                     raise e(msg)
                 else:
-                    logging.warning(msg)
+                    logging.debug(msg)
```

Comment:
I think these should probably still be warnings (even if it does fill up the log occasionally with faucet errors) as it's useful to see and I worry we'll forget that it exists otherwise